Adjusted to support multitenancy

danjoa opened the pull request.

nkaputnik commented on cds-plugin.js:
```diff
@@ -30,196 +33,220 @@ const addSideEffects = (actions, flag, element) => {
-            });
-          } else {
-            keys.push(e);
+      if (!entity['@changelog.disable_assoc']) { // REVISIT: why do we need that annotation?
```
This is a flag to disable automatic association definition. This is for people who want to change-track things, but don't want the change-log to be attached to the entity itself. They rather build a central Admin UI to view all changes.
One of the reasons is also, that ALL changes are visible through this association, regardless of who did them, something some stakeholders see as security issue

danjoa commented:
I fixed the failing tests by ...
1. Parked adding the changes view manually 
2. Added the `else` again to [line 209](https://github.com/cap-js/change-tracking/blob/ea8876769ab09f4ffb47ef048b6336d70b22ee89/cds-plugin.js#L209)
